### PR TITLE
lvm: add support for --data to be a raw device or partition

### PR DESCRIPTION
**Pull request by andrewschoen**

### CODE REVIEW

**Comment by andrewschoen** [comment]
jenkins test dev-ansible2.3-lvm_osds

**Comment by andrewschoen** [comment]
jenkins test dev-ansible2.4-lvm_osds

**Comment by andrewschoen** [comment]
This is for https://github.com/ceph/ceph/pull/18924

File: roles/ceph-osd/tasks/scenarios/lvm.yml [hunk 1]
```diff
@@ -7,13 +7,16 @@
   changed_when: False
   check_mode: no
 
+# --data as an lv
```
Comment (leseb) [inline roles/ceph-osd/tasks/scenarios/lvm.yml]: leftover?

File: roles/ceph-osd/tasks/scenarios/lvm.yml [hunk 2]
```diff
@@ -74,7 +81,88 @@
   when:
     - osd_objectstore == 'bluestore'
     - item.wal is not defined
+    - item.data_vg is defined
     - item.wal_vg is not defined
     - item.db is not defined
     - item.db_vg is not defined
     - "'{{ item.data_vg }}/{{ item.data }}' not in ceph_volume_lvm_list.stdout"
+
+# --data as a raw device or partition
```
Comment (leseb) [inline roles/ceph-osd/tasks/scenarios/lvm.yml]: ditto?

**Comment by andrewschoen** [comment]
@leseb I've removed the comments.

**Pull request merged**